language/golang/golang.go

```diff
@@ -59,6 +59,13 @@ func (l *Language) Files(log *log.Logger, repositoryPath string) (filePaths []st
 	return filePaths, nil
 }
 
+// TestFilePath returns the file path of a test file given the corresponding file path of the test's source file.
+// The file paths may be relative.
+func (l *Language) TestFilePath(filePath string) (testFilePath string) {
```

Comment:
Please add the repository path as well. The file path of Java projects is dependent on the projects path.

Reply:
Mhm or we change the docstring that the filepath always needs to be relative to the project root?

Reply:
but just adding the project root is probably fine as well

Reply:
All our parameters are always relative to their context. So would not mention that in the doc